### ISSUE TITLE
make publication pages for sessions with triage status Browser datasets, hosted

### ISSUE THREAD

**Issue by Antonialock** [pombase/website#1350]
I think
"curatable" and "Browser datasets, to host" get publication pages now, extend this to
"Browser datasets, hosted"

how about 
reviews
WT features
?

**Comment by ValWood** [comment]
I agree, reviews and methods should both get pages. We will need to change the message on these pages so it doesn't say:

This publication has no annotations in PomBase because it hasn't been curated yet. (Please contact us if you would like to contribute annotations for this paper.)

making use of the "method or reagent" or "review" classifier

there might  also be info in the Pubmed keywords what might be useful for searching (often includes gene names and subject")
